util/texlive/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM debian:jessie
+MAINTAINER nate.brennand@clever.com
+# Install latex related tooling:
+RUN apt-get update -y && \
+    apt-get install ca-certificates -y && \
```

Comment:
There's some officially maintained base images that have some of this installed already: https://hub.docker.com/r/library/buildpack-deps/

I like the 50mb xenial-curl image: https://github.com/docker-library/buildpack-deps/blob/master/xenial/curl/Dockerfile